webapp/tests/Unit/Controller/API/ProblemControllerAdminTest.php

```diff
@@ -18,7 +18,7 @@ protected function setUp(): void
     {
         // When queried as admin, extra information is returned about each problem.
         $this->expectedObjects[1]['test_data_count'] = 1;
-        $this->expectedObjects[2]['test_data_count'] = 3;
+        $this->expectedObjects[2]['test_data_count'] = 1+3;
```

Comment:
Can you add a comment why this is not 4 but 1+3?

Reply:
Done